### ISSUE TITLE
[PR/Linux] Add Linux support

### ISSUE THREAD

**Issue by WillXProject** [leminlimez/CowabungaLite#5]
Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch

**Comment by leminlimez** [comment]
if u get linux support working, create a pr and i will look into it

**Comment by WillXProject** [comment]
It may need loads of changing, so I'll do a new branch and see how it goes

**Comment by Yaksinikos** [comment]
may adding linux support in a indirect way, using wine?

**Comment by olivertzeng** [comment]
> may adding linux support in a indirect way, using wine?

Nope, wine never worked for me

**Comment by olivertzeng** [comment]
> may adding linux support in a indirect way, using wine?

~~and I'm only fifteen yo~~

**Comment by BomberFish** [comment]
Linux support shouldn't be hard, it only require some small tweaks to the codebase to work properly.

**Comment by olivertzeng** [comment]
oh ok

**Comment by LunarN0v4** [comment]
> Linux support shouldn't be hard, it only require some small tweaks to the codebase to work properly.

yeah, if school wasn't constantly asking me to do homework every minute of my fucking afternoons and weekends, it'd be easy

**Comment by olivertzeng** [comment]
oh well good luck with your assignments...

**Comment by LunarN0v4** [comment]
> oh well good luck with your assignments...

ty >w<

**Comment by olivertzeng** [comment]
owo
Really wish Linux support will arrive soon.

**Comment by LunarN0v4** [comment]
> owo Really wish Linux support will arrive soon.

same, then again, I've been able to go without a jailbreak for a while, so I'm phasing it out rn

**Comment by olivertzeng** [comment]
Same here, waiting for A15 iOS 16.5 support

**Comment by LunarN0v4** [comment]
> Same here, waiting for A15 iOS 16.5 support

nah, I'm just not jailbreaking at all anymore

**Comment by LunarN0v4** [comment]
> may adding linux support in a indirect way, using wine?

Wine doesn't have access to /dev/ or devices and no access to usbmuxd

**Comment by leminlimez** [comment]
linux cli support should be really easy to add, you just need to replace the windows binaries with linux for libimobiledevice

**Comment by olivertzeng** [comment]
> Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch

As an Arch end user, I think flatpaks are better than appimage

**Comment by LunarN0v4** [comment]
> > Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch
> 
> As an Arch end user, I think flatpaks are better than appimage

I also use Arch, but it highly depends on what type of person you are lol

**Comment by olivertzeng** [comment]
> > > Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch
> > 
> > 
> > As an Arch end user, I think flatpaks are better than appimage
> 
> I also use Arch, but it highly depends on what type of person you are lol

ok! cool! oh yeah is the linux version done?

**Comment by LunarN0v4** [comment]
> > > > Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch
> > > 
> > > 
> > > As an Arch end user, I think flatpaks are better than appimage
> > 
> > 
> > I also use Arch, but it highly depends on what type of person you are lol
> 
> ok! cool! oh yeah is the linux version done?

no?

**Comment by olivertzeng** [comment]
> > > > > Creating a Linux AppImage is surprisingly simple, however, I need to know whether it would be better to have my own fork or pull request it to a new branch
> > > > 
> > > > 
> > > > As an Arch end user, I think flatpaks are better than appimage
> > > 
> > > 
> > > I also use Arch, but it highly depends on what type of person you are lol
> > 
> > 
> > ok! cool! oh yeah is the linux version done?
> 
> no?

ok

**Comment by olivertzeng** [comment]
@LunarN0v4 where is your linux version of cowabunga lite? I want to see what it has, really want to learn how i can port codes to linux

**Comment by olivertzeng** [comment]
> linux cli support should be really easy to add, you just need to replace the windows binaries with linux for libimobiledevice

Is qt hard to develop? I know qt but have never ever made a gui before.
